proxy/proxy.go

```diff
@@ -22,9 +22,10 @@ type Proxy struct {
 	client         *docker.Client
 	withDNS        bool
 	dockerBridgeIP string
+	forceIPAM      bool
 }
 
-func NewProxy(targetURL string, withDNS bool) (*Proxy, error) {
+func NewProxy(targetURL string, withDNS, forceIPAM bool) (*Proxy, error) {
```

Comment:
why not `withIPAM`, and `--with-ipam`, for consistency with `withDNS`/`--with-dns`?